capstone/src/main/resources/static/style.css

```diff
@@ -128,7 +128,7 @@ img {
   width: 40%;
 }
 
-.profile-form input[type=text], .profile-form select {
+.profile-form input[type=text], .profile-form input[type=datetime-local], .profile-form input[type=number], .profile-form textarea, .profile-form select {
```

Comment:
It's conventional in CSS to list each separate selector path on a different line, for readability:
```
.profile-form input[type=text],
.profile-form input[type=datetime-local],
.profile-form input[type=number],
.profile-form textarea, .profile-form select
```